demoapp/extract_mgl_sections.py

```diff
@@ -0,0 +1,319 @@
+import pandas as pd
```

Comment:
Thanks so much for making this!  Can you add a doclevel comment to explain the purpose?

Reply:
I think this comment has not yet been addressed.